client/post-editor/post-editor.jsx

```diff
@@ -569,7 +573,10 @@ const PostEditor = React.createClass( {
 
 		this.saveRawContent();
 		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
-		actions.edit( { content: this.refs.editor.getContent() } );
+		const edits = merge( {}, this.props.edits, {
```

Comment:
Couple thoughts here:

Could we get away with a shallow clone? That opens up possibility of mutating the original reference on nested properties (terms, discussion), but I feel that's unlikely to happen.

```js
const edits = {
	...this.props.edits,
	content: this.refs.editor.getContent()
};
```

Otherwise, we could make this ever so slightly more efficient by merging the prop edits into the constructed object directly.

```js
const edits = merge( {
	content: this.refs.editor.getContent()
}, this.props.edits );
```

(Same goes for below where we're created a new merged object with `status`)